test/Common/SupportedLanguagesTestAttribute.cs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using Xunit.Sdk;
+using System.Reflection;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Tests.Common
+{
+    /// <summary>
+    /// Base attribute class for adding language parameter data to tests. This allows
+    /// tests to be run against all the supported languages list that are specified in the SupportedLanguages
+    /// list of this class (ex: CSharp, JavaScript)
+    /// </summary>
+    public abstract class LanguageTestAttribute : DataAttribute
+    {
+        // add all the list of supported languages.
+        protected enum SupportedLanguages
+        {
+            CSharp
+        }
+        protected readonly Dictionary<string, List<List<object>>> testData = new Dictionary<string, List<List<object>>>();
+        protected readonly string _propertyName = "data";
+
+        /// <inheritDoc />
+        public override IEnumerable<object[]> GetData(MethodInfo testMethod)
+        {
+            if (testMethod == null) { throw new ArgumentNullException(nameof(testMethod)); }
```

Comment:
Are you copying this code from somewhere? It seems overly complex for what we need to do - especially with all the JSON parsing. But let's address the other issues and then look at this closer if this still ends up being necessary.

Reply:
I referenced a blog that extends the DataAtrribute but this step is specific to us, the blog reads data from file so just parsing and returning. But in our case it's in a list so added the serialization before I parse.